src/com/twu/biblioteca/Printer.java

```diff
@@ -65,4 +67,16 @@ private static void printLibraryItems(ArrayList<? extends LibraryItem> libraryIt
         }
         System.out.println();
     }
+
+    public static void askMovieName() {
+        System.out.println("Type the name of the movie you want to checkout");
+    }
+
+    public static void notifyUserSuccessfulMovieCheckout() {
```

Comment:
Implement a method `notifyUserSuccessfulCheckout(String element)` instead of having 2 duplicate methods.